Templates/AppSource App/.github/workflows/CICD.yaml

```diff
@@ -245,6 +245,17 @@ jobs:
           settingsJson: ${{ env.Settings }}
           secretsJson: ${{ env.RepoSecrets }}
           buildMode: ${{ matrix.buildMode }}
+      
+      - name: Sign
+        if: env.doNotSignApps == 'False' && env.keyVaultCodesignCertificateName != ''
+        id: sign
+        uses: microsoft/AL-Go-Actions/Sign@main
+        with:
+          shell: ${{ needs.Initialization.outputs.githubRunnerShell }}
+          azureCredentialsJson: ${{ secrets.AZURE_CREDENTIALS }}
```

Comment:
Isn't it better to pass AZURE_CREDENTIALS as a secret that action can read, than pass it as a parameter?

Reply:
Yeah, I guess it's more or less the same.